views/default/resources/account/change_password.php

```diff
@@ -32,7 +32,6 @@
 $content = elgg_view_form('user/changepassword', array('class' => 'elgg-form-account'), $params);
 
 if (elgg_get_config('walled_garden')) {
-	elgg_load_css('elgg.walled_garden');
 	$body = elgg_view_layout('walled_garden', array('content' => $content));
```

Comment:
so passing title to layout would draw a title (just as the regular shell does) and completely drop the walled garden module as it is just normal content? all up for it 👍